databases/week1/hw-week1.sql

```diff
@@ -0,0 +1,47 @@
+
+select * from task ;
+-- task 1
+select count(*) as tasks_count from task;
+
+-- task 2
+select title from task
+where due_date is null;
+
+ -- task 3
+select title from task
+where status_id = 3;
+
+-- task 4
+select title from task
+where status_id = 1 and 2;
```

Comment:
the only data we get back here is, WHERE status_id = 1.

Reply:
where status_id = 1 and  status_id = 2;